### PR TITLE
Feat/include new profiles

### PR DESCRIPTION
**Pull request by redur**
Added new set of borehole profiles, and with it, multilingual support.

- Adjusted GroundTruth to work with new format. --> old format will not work any more. There is a functionality in the notebook that converts old ground_truth.json into the new one.
- Added language detection.
- Translated including and excluding expression into french.
- Also comes with the EDA based on a different branch. Probably should be dropped here. Kept for the conversion script and the generation of test/train/eval splits. Can be dropped after the review.

### CODE REVIEW

File: src/stratigraphy/util/language_detection.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    Returns:
+        str: The extracted and processed text.
+    """
+    text = ""
+    for page in doc:
+        text += page.get_text()
+    text = text.replace("\n", " ")
+
+    # remove all numbers and special characters from text
+    return "".join(e for e in text if (e.isalnum() or e.isspace()) and not e.isdigit())
```
Comment (stijnvermeeren-swisstopo) [inline src/stratigraphy/util/language_detection.py:23]: Does this keep accented letter such as öäüéàè that might be important for detecting German/French, or only ASCII?

**Review by stijnvermeeren-swisstopo — COMMENTED** [review]
I would indeed remove the notebook and not merge it into the main code base.

What's the easiest way for us to share our train/validation/test splits? Do you recommend that I execute the notebook myself, using the exact same random seed, or is there a more convenient way?

File: src/stratigraphy/util/language_detection.py (hunk 1 above)
Comment (redur) [reply src/stratigraphy/util/language_detection.py:23]: Good question. I'll have to check.

Comment (redur) [reply src/stratigraphy/util/language_detection.py:23]: öäüéàè and the like are kept.

**Review by stijnvermeeren-swisstopo — APPROVED** [review]
LGTM.

For now, it's good to have the convert_ground_script here for reproducibility, but in future we can probable remove that script again, as we won't need to deal with the old format at all anymore?

**Comment by redur** [comment]
> LGTM.
> 
> For now, it's good to have the convert_ground_script here for reproducibility, but in future we can probable remove that script again, as we won't need to deal with the old format at all anymore?

I agree with this statement.

**Pull request merged**